src/options/options_handler.cpp

```diff
@@ -198,27 +198,27 @@ void OptionsHandler::checkBitblastMode(std::string option, BitblastMode m)
   }
   else if (m == BitblastMode::EAGER)
   {
-    if (!options::bitvectorToBool.wasSetByUser())
+    if (!Options::current().wasSetByUser(options::bitvectorToBool))
     {
-      options::bitvectorToBool.set(true);
+      Options::current().set(options::bitvectorToBool, true);
     }
   }
 }
 
 void OptionsHandler::setBitblastAig(std::string option, bool arg)
 {
   if(arg) {
-    if(options::bitblastMode.wasSetByUser()) {
+    if(Options::current().wasSetByUser(options::bitblastMode)) {
       if (options::bitblastMode() != options::BitblastMode::EAGER)
       {
         throw OptionException("bitblast-aig must be used with eager bitblaster");
       }
     } else {
-      options::BitblastMode mode = stringToBitblastMode("", "eager");
-      options::bitblastMode.set(mode);
+      options::BitblastMode mode = stringToBitblastMode("eager");
+      Options::current().set(options::bitblastMode, mode);
     }
-    if(!options::bitvectorAigSimplifications.wasSetByUser()) {
-      options::bitvectorAigSimplifications.set("balance;drw");
+    if(!Options::current().wasSetByUser(options::bitvectorAigSimplifications)) {
+      Options::current().set(options::bitvectorAigSimplifications, "balance;drw");
```

Comment:
Shouldn't this default actually be specified in the toml file, not here?

Reply:
Not sure, you tell me ;-)
But given how the relevant options are named and the help message of these options, I'd agree...